umbraco/Umbraco-CMS
# V8: Handle mixed variance when creating content templates from content

### Prerequisites


### Description

When creating content templates from content (assuming #4782 is applied so you can actually do that) with mixed property variance (some properties culture variant, some not), things blow up with the following message:

![image](https://user-images.githubusercontent.com/7405322/53512982-5d032900-3ac4-11e9-92df-295e93d038be.png)

It happens because the content template creation assumes that the original content item is either completely culture variant or completely culture *in*variant.

This PR fixes the issue.

#### Testing this PR
First of all you can't test this PR until #4782 is merged in.

1. Create a content template from some content with mixed culture variance.
2. Verify that nothing blows up.
3. Verify that the template is indeed created and that the culture variant and *in*variant properties on the template are mapped correctly from the original content item.
